README.md

```diff
@@ -120,14 +120,26 @@ Explicit steps for setting up CCM in a development environment.
     2. In the "Configuration Type" menu, select "By Client ID".
     3. In the "Client ID" field, paste in the same ID number that was added to `.env` above.
     4. When prompted to verify the ID for the tool, click the "Install" button.
+#### Database Migrations
```

Comment:
@pushyamig, I tried to help clarify and format some of this documentation (I hope that's okay). Please also review this section to make sure that indentation is consistent with the rest of the `README.md`. I think we could use unordered bullets here instead, but it's okay this way. I tried to break up longer lines, since that's what we've been doing so far; doing so doesn't have any effect on the rendered Markdown and it makes editing the text easier.

Reply:
I checked for indentation and it look ok.